spec/json/matchers/match_response_schema_spec.rb

```diff
@@ -22,6 +22,23 @@
     expect(response_for({})).not_to match_response_schema("foo_schema")
   end
 
+  it "accepts options for the validator" do
+    create_schema("foo_schema", {
+      "type" => "object",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.